quilt/test/test_install.py

```diff
@@ -125,3 +125,15 @@ def _mock_package(self, package, pkg_hash, contents, hashes):
     def _mock_s3(self, pkg_hash, contents):
         s3_url = 'https://example.com/%s' % pkg_hash
         self.requests_mock.add(responses.GET, s3_url, contents)
+
+# execute outside of mock scope so we get real network
```

Comment:
how about a `pytest --integration` flag; off by default? and then we can decorate integration tests and skip without it.